### PR TITLE
Flutter stake tokens guide

### PR DESCRIPTION
**Pull request by emmaodia**

### CODE REVIEW

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+cd fuse_wallet_app
+```
+
+Next, add the Fuse Wallet SDK to your project. Open your pubspec.yaml and add the following line under dependencies:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  fuse_wallet_sdk: latest_version
```
Comment (NikolayRn) [inline docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx:28]: This doesnt worked for me, replace it with actual latest version

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+    initFuseSDK();
+  }
+
+  Future<void> initFuseSDK() async {
+    fuseSDK = await FuseSDK.init(apiKey, privateKey);
+    setState(() {
+      // If the SDK returns wallet details or needs to display the address
+      _address = fuseSDK.wallet.getSender();
+    });
+    print("Fuse SDK initialized successfully!");
```
Comment (NikolayRn) [inline docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx:82]: Got errors here 
Screenshot 2024-05-30 at 19 00 00

**Review by NikolayRn — CHANGES_REQUESTED** [review]
I faced errors and didnt completed tutorial, please fix

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx (hunk 2 above)
Comment (emmaodia) [reply docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx:82]: Fixed.

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx (hunk 1 above)
Comment (emmaodia) [reply docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx]: Fixed.

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+
+```dart
+import 'package:flutter/material.dart';
+import 'package:fuse_wallet_sdk/fuse_wallet_sdk.dart';
+
+void main() {
+  runApp(MyApp());
+}
+
+class MyApp extends StatelessWidget {
```
Comment (NikolayRn) [inline docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx:50]: I copied updated code and faced this errors: 
Screenshot 2024-06-02 at 17 27 54
Screenshot 2024-06-02 at 17 27 57

**Review by NikolayRn — CHANGES_REQUESTED** [review]
See the comments

File: docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx (hunk 3 above)
Comment (emmaodia) [reply docs/fuse-box/tutorials/mobile-stake-tokens-guide.mdx:50]: Fixed.

**Comment by LiorAgnin** [comment]
@emmaodia FYI, This API will probably be deprecated soon.

**Pull request closed without merging**